src/pages/[platform]/build-a-backend/data/aws-appsync-apollo-extensions/index.mdx

```diff
@@ -38,32 +37,42 @@ To learn more about Apollo, see https://www.apollographql.com/docs/kotlin.
 
 </InlineFilter>
 
-## Features
+## Install the AWS AppSync Apollo Extensions library
 
-AWS AppSync Apollo Extensions provide AWS AppSync authorizers to be used with the Apollo client to make it simple to apply the correct authorization payloads to your GraphQL operations.
+<InlineFilter filters={["android"]}>
 
-The Amplify library provides components to facilitate configuring the authorizers with Apollo client by providing configuration values to connect to your Amplify Data backend.
+<BlockSwitcher>
 
-### Install the AWS AppSync Apollo Extensions library
+<Block name=" Amplify Data">
```

Comment:
We need something to explain what the difference between these tabs are. The comments in the first code block explain it but those are easy to miss.

Reply:
@mattcreaser Are you suggesting explaining this once in the intro, or modifying the names of the tabs?

Reply:
Actually, if we go with "With Amplify and "Without Amplify" that clears it up immediately.